plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java

```diff
@@ -329,15 +333,15 @@ else if (args.getString(1).equalsIgnoreCase("scripts")) {
                 if (ScriptHelper.hadError()) {
                     Messaging.sendError(sender, "There was an error loading your scripts, check the console for details!");
                 }
-                Messaging.sendError(sender, "'/denizen reload scripts' is the old way of doing things ... use '/ex reload' instead!");
+                Messaging.sendError(sender, "'/denizen reload scripts' is deprecated, use '/ex reload' instead.");

```

Comment:
Incorrect change

Reply:
Understood. I thought that deprecated would've been the right wording, will change.